lib/teracy-dev/extension/manager.rb

```diff
@@ -52,13 +53,22 @@ def sync(extension)
           "path" => path
         })
         sync_existing = extension['path']['lookup'] == DEFAULT_EXTENSION_LOOKUP_PATH
-        Location::Manager.sync(extension['location'], sync_existing)
+
+        if Util.cmdRequireSync?
```

Comment:
move this condition into Location::Manager.sync instead, add optional argument force=false; if force = true and Util.cmdRequireSync? is false => continue syncing

`
Location::Manager.sync(location, sync_existing, force=False)
`